FORK_SCHEMA.md

```
@@ -0,0 +1,472 @@
+# Fork Schema: Auto-Claude
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)